src/lsptoolshost/onAutoInsert.ts

```diff
@@ -40,7 +40,15 @@ export function registerOnAutoInsert(languageServer: RoslynLanguageServer) {
 
             source.cancel();
```

Comment:
I don't see source being used anywhere before this point -- can we simplify this?